runtime/builtin_types.go

```diff
@@ -827,3 +861,18 @@ func zipLongest(f *Frame, args Args) ([][]*Object, *BaseException) {
 	}
 	return result, nil
 }
+
+func round(number float64, places int) float64 {
+	var round float64
```

Comment:
This seems kind of crazy but then I read what CPython does and it's even more wild: https://github.com/python/cpython/blob/2.7/Objects/floatobject.c#L1092

My concern is that there will be subtle differences in behavior here with respect to CPython and it will only surface as part of divergent behavior in a complex program that will be difficult to debug. I would recommend either:

1. Write a dead-simple implementation for the purposes of unblocking unittest (e.g. Round() from [this gist comment](https://gist.github.com/DavidVaini/10308388#gistcomment-1391788))

2. Write a comprehensive implementation based on what's in floatobject.c linked above.

The PR as written is neither simple nor comprehensive so it's liable to end up causing us debugging headaches down the road.